app/javascript/src/components/Authentication/Signup.jsx

```diff
@@ -51,65 +56,58 @@ const Signup = ({ history }) => {
         <h2 className="mb-5 text-3xl font-extrabold text-center text-gray-800">
           Signup
         </h2>
-
-        <form
-          className="w-full p-8 space-y-6 bg-white border rounded-md shadow"
+        <Formik
+          initialValues={{
```

Comment:
Initial values do not change and the object can be stored as constant in a separate constants file along with the validationSchema.